src/lib/CometVisuClient.js

```diff
@@ -15,21 +15,18 @@
  * You should have received a copy of the GNU General Public License along
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
- *
- * @module CometVisuClient 
- * @title  CometVisu CometVisuClient 
  */
 
 
 /**
  * The JavaScript library that implements the CometVisu protocol.
  *
- * @title CometVisu-Client
- * @exports ComentVisuVlient
- * @requires jQuery
+ * @module lib/CometVisuClient
+ * @exports ComentVisuClient
+ * @requires dependencies/jquery
  * @author Christan Mayer
  * @author Tobias Bräutigam
- * @since 2010
+ * @since 0.5.3 (initial contribution) 0.10.0 (major refactoring)
```

Comment:
No not sure, I just took the first version after the year 2010 (as it was specified as the year of the contribution). Referring to the available releases on github I could say that is has been part of 0.5.1 (which is the oldest one there), I don't know/remember which one was the first public release. So any suggestions welcome.